[2023-12-18 11:00:01,826] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: dag_with_python_weather_v01.fill 2023-12-18T10:00:00+00:00 [queued]>
[2023-12-18 11:00:01,850] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: dag_with_python_weather_v01.fill 2023-12-18T10:00:00+00:00 [queued]>
[2023-12-18 11:00:01,850] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2023-12-18 11:00:01,851] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2023-12-18 11:00:01,851] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2023-12-18 11:00:01,863] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): fill> on 2023-12-18T10:00:00+00:00
[2023-12-18 11:00:01,872] {standard_task_runner.py:52} INFO - Started process 82528 to run task
[2023-12-18 11:00:01,878] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'dag_with_python_weather_v01', 'fill', '2023-12-18T10:00:00+00:00', '--job-id', '149', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_python_weather.py', '--cfg-path', '/tmp/tmp8yuxjhjm', '--error-file', '/tmp/tmp8tr4xzel']
[2023-12-18 11:00:01,881] {standard_task_runner.py:77} INFO - Job 149: Subtask fill
[2023-12-18 11:00:01,942] {logging_mixin.py:104} INFO - Running <TaskInstance: dag_with_python_weather_v01.fill 2023-12-18T10:00:00+00:00 [running]> on host 5f3722f1fc3b
[2023-12-18 11:00:02,002] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=ggoni
AIRFLOW_CTX_DAG_ID=dag_with_python_weather_v01
AIRFLOW_CTX_TASK_ID=fill
AIRFLOW_CTX_EXECUTION_DATE=2023-12-18T10:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-12-18T10:00:00+00:00
[2023-12-18 11:00:06,957] {python.py:118} INFO - Done. Returned value was: None
[2023-12-18 11:00:06,975] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=dag_with_python_weather_v01, task_id=fill, execution_date=20231218T100000, start_date=20231218T110001, end_date=20231218T110006
[2023-12-18 11:00:07,029] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-12-18 11:00:07,066] {local_task_job.py:146} INFO - Task exited with return code 0
